t/local_ingest.t

```diff
@@ -162,9 +162,16 @@ describe "HTFeed::PackageType::Simple" => sub {
       my $exiftool = Image::ExifTool->new();
       $exiftool->ExtractInfo("$tmpdirs->{ingest}/lossless_jp2_with_xmp/00000001.jp2");
       is($exiftool->GetValue("XMP-tiff:Make"),"Test scanner make");
-
     };
 
+    it "does not lose artist when compressing a bitonal tiff" => sub {
+	my $volume = unpacked_volume("bitonal_tiff");
```

Comment:
This tab indentation throws off my editor. My recommendation may apply to other code in the PR but it is especially noticeable here. AFAICT this block should be indented with two tabs but is only one, and IMHO (at the risk of igniting a flame war) there should be no tabs at all, just explicit spaces -- whatever the norm is for HathiTrust Perl, seems to be a toss-up between 2 and 4 spaces.